cargo-apk/src/apk.rs

```diff
@@ -167,7 +167,11 @@ impl<'a> ApkBuilder<'a> {
                 .map(|path| path.as_path())
                 .collect::<Vec<_>>();
 
-            apk.add_lib_recursively(&artifact, *target, libs_search_paths.as_slice())?;
+            // allow using cargo-apk for building libraries in ci that don't produce a cdylib
```

Comment:
You can probably leave out the `in ci` part or put it in parentheses - I take it this is useful on a host system too.